main.py

```diff
@@ -3,27 +3,67 @@
 """
 
 import sys
-from PySide6.QtWidgets import QApplication, QMainWindow
+from PySide6.QtWidgets import QApplication, QMainWindow, QStackedWidget
 from dunwoody_disaster.views.fightScreen import FightScreen
-from dunwoody_disaster.views.collectLootScreen import CollectLootScreen
 from dunwoody_disaster.CharacterFactory import CharacterFactory
-from dunwoody_disaster import Item
+
+from dunwoody_disaster.views.StartMenu import StartMenu
+from dunwoody_disaster.views.MapScreen import MapScreen
 
 
 class MainWindow(QMainWindow):
     def __init__(self):
         super().__init__()
-        self.setWindowTitle("Pokemon-like Game")
-        self.setGeometry(100, 100, 800, 600)
+        self.setWindowTitle("Dunwoody-Disaster")
+        # self.setGeometry(100, 100, 800, 600)
         self.setStyleSheet("background-color: #2f2f2f;")
 
-        player1 = CharacterFactory.createTestChar()
-        player2 = CharacterFactory.createTestChar()
-        if 1 == 2:  # this is here to clear lint warnings
-            self.setCentralWidget(FightScreen(player1, player2))
-        else:
-            items = Item.weapons + Item.armors
-            self.setCentralWidget(CollectLootScreen(player1, items))
+        self.player1 = CharacterFactory.createTestChar()
+        self.testChar = CharacterFactory.createTestChar()
+        # Enemies
+        self.courtChar = CharacterFactory.createTestChar()
+        self.lectChar = CharacterFactory.createTestChar()
+        self.phyLabChar = CharacterFactory.createTestChar()
+        self.sciLabChar = CharacterFactory.createTestChar()
+
+        self.startMenu = StartMenu()
+        self.startMenu.startButton.clicked.connect(self.showMapScreen)
+        self.mapScreen = MapScreen(self.EnterFight)
+        self.fightScreen = FightScreen(self.player1, self.testChar)
+
+        self.stack = QStackedWidget()
+        self.stack.addWidget(self.startMenu)
+        self.stack.addWidget(self.mapScreen)
+        self.stack.addWidget(self.fightScreen)
+
+        # Set the stacked widget as the central widget of the main window
+        self.setCentralWidget(self.stack)
+
+    def showMapScreen(self):
+        self.stack.setCurrentIndex(1)
+
+    def EnterFight(self):
+        """
+        Enter fight screen by pointing stack at fight screen.
+        This will need to be changed to set the proper opponent per setting. Index 2 is the fight screen.
+        """
+        self.stack.setCurrentIndex(2)
+        self.player1 = self.player1
+        self.player2 = self.courtChar
```

Comment:
You need to set the fightScreen character variables equal to the characters you want. Currently setting the local players to other objects.